src/epplibwrapper/tests/test_client.py

```diff
@@ -255,3 +261,101 @@ def side_effect(*args, **kwargs):
             mock_close.assert_called_once()
             # send() is called 5 times: send(login), send(command) fail, send(logout), send(login), send(command)
             self.assertEquals(mock_send.call_count, 5)
+
+    def test_send_command_close_failure_recovers(self):
+        """Test when the .close on a connection fails and a .send follows suit.
```

Comment:
**blocking question**: maybe it's friday and I'm just reading this all wrong, but aren't you really testing if a connection fails entirely?